xrpl/utils/get_nftoken_id.py

```diff
@@ -97,6 +97,9 @@ def has_nftoken_page(node: Node) -> bool:
 
     affected_nodes = [node for node in meta["AffectedNodes"] if has_nftoken_page(node)]
 
+    if len(affected_nodes) == 0:
```

Comment:
Is it typical to throw a exception and not just return None.  This would be a normal occurance in cases where there is a tec code.

Reply:
I think it'd be more difficult to identify a bug if it returns `None` instead of erroring.

Reply:
I agree with Mayukha's approach of throwing an error because it indicates invalid user input; this justifies throwing an error. We should probably make it consistent with JS to throw an error too.

Reply:
However, it's also valid to have a failed transaction and not throw an error. So now I'm leaning more towards not throwing an error and keeping things consistent 😄